maven-release-manager/src/main/java/org/apache/maven/shared/release/phase/AbstractRunGoalsPhase.java

```diff
@@ -71,11 +72,12 @@ protected ReleaseResult execute( ReleaseDescriptor releaseDescriptor, ReleaseEnv
             String goals = getGoals( releaseDescriptor );
             if ( !StringUtils.isEmpty( goals ) )
             {
-                logInfo( result, "Executing goals '" + goals + "'..." );
-                if ( logArguments && ( additionalArguments != null ) )
+                logInfo( result, "Executing goals '" + buffer().strong( goals ) + "'..." );
+                if ( logArguments )
                 {
                     // logging arguments may log secrets: should be activated only on dryRun
-                    logInfo( result, "    with additional arguments: " + additionalArguments );
+                    logInfo( result, "    with additional arguments: "
+                        + ( additionalArguments == null ? "(none)" : additionalArguments ) );
```

Comment:
Stupid question: Can this string be empty?

Reply:
yes, it happens

Reply:
Then the test should include null and empty.